### PR TITLE
Add initial GitHub Issue Form

### PR DESCRIPTION
**Pull request by frenck**
## Proposed change

First attempt to add an GitHub Issue form to this repository.

I've picked this repository to go first, of the options we have, this has the lowest amount of issue traffic. Let's see how it looks, create some issues, see if we like it, learn a bit; before we roll it out to other repos.

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (which adds functionality to the supervisor)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] Local tests pass. **Your PR cannot be merged unless tests pass**
- [ ] There is no commented out code in this PR.
- [ ] I have followed the [development checklist][dev-checklist]
- [ ] The code has been formatted using Black (`black --fast supervisor tests`)
- [ ] Tests have been added to verify that the new code works.

If API endpoints of add-on configuration are added/changed:

- [ ] Documentation added/updated for [developers.home-assistant.io][docs-repository]

[dev-checklist]: https://developers.home-assistant.io/docs/en/development_checklist.html
[docs-repository]: https://github.com/home-assistant/developers.home-assistant

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+        - Other (e.g., Raspbian/Raspberry Pi OS/Fedora)
+  - type: input
+    attributes:
+      label: What is the version of your installed operating system?
+      required: true
+      placeholder: 5.10
+      description: Can be found in the Supervisor panel -> System tab.
+  - type: input
+    attributes:
+      label: What is version of Home Assistant Core is installed?
```
Comment (MartinHjelmare) [inline .github/ISSUE_TEMPLATE/bug_report.yml:47]:
```suggestion
      label: What version of Home Assistant Core is installed?
```

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+        If you don't know, you can find it in: Configuration panel -> Info.
+      choices:
+        - Home Assistant OS
+        - Home Assistant Supervised
+  - type: dropdown
+    attributes:
+      label: Which operating system are you running on?
+      required: true
+      choices:
+        - Home Assistant Operating System (HassOS)
```
Comment (ludeeus) [inline .github/ISSUE_TEMPLATE/bug_report.yml:36]:
```suggestion
        - Home Assistant Operating System (HAOS)
```

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+        - Debian
+        - Other (e.g., Raspbian/Raspberry Pi OS/Fedora)
```
Comment (ludeeus) [inline .github/ISSUE_TEMPLATE/bug_report.yml:38]:
```suggestion
```
You are instructed at the beginning of the file to not use this for unsupported, I don't think this is needed.

Comment (frenck) [reply .github/ISSUE_TEMPLATE/bug_report.yml:38]: I actually added it on purpose. For 2 reasons:

1. People don't care about the comment.
2. It's a trap.
3. Please don't know Raspbian and Debian are different (Raspabian Debian (Buster) is part of their OS description).

By making this more explicit, it becomes more clear.

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 2 above)
Comment (frenck) [reply .github/ISSUE_TEMPLATE/bug_report.yml:36]: HassOS is still displayed. I did not try to shorten the installation method, instead, it was a reference to the operating system itself (which still is displayed as HassOS)

Comment (ludeeus) [reply .github/ISSUE_TEMPLATE/bug_report.yml:36]: Where is it displayed like that?

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 3 above)
Comment (ludeeus) [reply .github/ISSUE_TEMPLATE/bug_report.yml:38]: Okey then :+1:

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 2 above)
Comment (frenck) [reply .github/ISSUE_TEMPLATE/bug_report.yml:36]: Not that many places, I agree. Let's remove the abbreviation completely then. Agree?

Comment (ludeeus) [reply .github/ISSUE_TEMPLATE/bug_report.yml:36]: That works too :+1:

**Review by ludeeus — APPROVED** [review]

**Pull request merged**